gatsby-config.js

```diff
@@ -50,104 +54,172 @@ module.exports = {
+                  path: "documentation/adobe-campaign-classic/api-reference"
+              },
+              {
+                  title: "Event reference",
+                  path: "documentation/adobe-campaign-classic/event-reference"
+              },
+              {
+                  title: "Release notes",
+                  path: "documentation/adobe-campaign-classic/release-notes"
+              }
+          ]
       },
       {
-        title: 'Migrating',
-        path: '/guides/migrating/'
+          title: "Adobe Journey Optimizer",
+          path: "/documentation/adobe-journey-optimizer",
+          pages: [
+              {
+                  title: "API reference",
+                  path: "documentation/adobe-journey-optimizer/api-reference"
+              },
+              {
+                  title: "Release notes",
+                  path: "documentation/adobe-journey-optimizer/release-notes"
+              }
+          ]
       },
       {
-        title: 'Overview',
-        path: '/support/',
-        header: true,
+        title: "Adobe Media Analytics",
+        path: "/documentation/adobe-media-analytics",
         pages: [
-          {
-            title: 'Help',
-            path: '/support/'
-          },
-          {
-            title: 'FAQ',
-            path: '/support/FAQ/'
-          },
-          {
-            title: 'How to contribute',
-            path: '/support/contribute/'
-          }
+            {
+                title: "API reference",
+                path: "documentation/adobe-media-analytics/api-reference"
+            },
+            {
+                title: "Migration",
+                path: "documentation/adobe-media-analytics/migration"
+            },
+            {
+                title: "Release notes",
+                path: "documentation/adobe-media-analytics/release-notes"
+            }
         ]
       },
       {
-        title: 'Community',
-        path: '/support/community/',
-        header: true,
-        pages: [
-          {
-            title: 'Information',
-            path: '/support/community/'
-          }
-        ]
+          title: "Adobe Target",
+          path: "/documentation/adobe-target",
+          pages: [
+              {
+                  title: "API reference",
+                  path: "documentation/adobe-target/api-reference"
+              },
+              {
+                  title: "Event reference",
+                  path: "documentation/adobe-target/event-reference"
+              },
+              {
+                  title: "Migration",
+                  path: "documentation/adobe-target/migration"
+              },
+              {
+                  title: "Release notes",
+                  path: "documentation/adobe-target/release-notes"
+              },
+              {
+                  title: "Deprecated APIs",
+                  path: "documentation/adobe-target/deprecated-apis"
+              }
+          ]
       }
     ]
```

Comment:
@calebk1 can we set the pathPrefix to something like: 
pathPrefix: process.env.PATH_PREFIX || '/aep-sdks/docs/'